src/plugins/orangehrmCorePlugin/Api/Rest/ValidationUniqueAPI.php

```diff
@@ -63,7 +63,8 @@ class ValidationUniqueAPI extends Endpoint implements ResourceEndpoint
         'LeaveType' => ['attributeName' => ['name']],
         'Employee' => ['attributeName' => ['employeeId']],
         'TerminationReason' => ['attributeName' => ['name']],
-        'Vacancy' => ['attributeName' => ['name']]
+        'Vacancy' => ['attributeName' => ['name']],
```

Comment:
@devishke-orange this seems a coupling right. anyway we did it simplest way. do you have any suggestions?